nifi-ignite-extensions/nifi-ignite-services/src/main/java/pl/touk/nifi/services/AbstractIgniteClientCache.java

```diff
@@ -22,6 +24,30 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    protected static final PropertyDescriptor BACKUPS = new PropertyDescriptor.Builder()
+            .displayName("Backups")
+            .name("backups-no")
+            .description("The number of partition backups")
+            .required(true)
```

Comment:
can we make it optional?

Reply:
I'm not convinced. I'd rather enforce user to explicitly pass 0 backups.